frontend/src/metabase/search/containers/SearchResultSection.tsx

```diff
@@ -0,0 +1,11 @@
+import type { WrappedResult } from "metabase/search/types";
+import Card from "metabase/components/Card";
+import { SearchResult } from "metabase/search/components/SearchResult";
+
+export const SearchResultSection = ({ items }: { items: WrappedResult[] }) => (
```

Comment:
Looks like this one belongs in `components`, not `containers`.

Reply:
I would say this is more `container` than `component` since it really just acts as a wrapper around the list of search results in `SearchApp`

Reply:
I'll need to take a better look at the actual structure of the `search` folder in general in the future

Reply:
Oh, I though containers are redux-connected components.